lte/gateway/python/integ_tests/s1aptests/test_no_auth_response_with_mme_restart.py

```diff
@@ -0,0 +1,112 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import time
+
+import s1ap_types
+import s1ap_wrapper
+from s1ap_utils import MagmadUtil
+
+
+# This test case is to test EPC behaviour by restarting mme
+# after authentication timer expires.
+
+
+class TestNoAuthResponseWithMmeRestart(unittest.TestCase):
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper(
+            stateless_mode=MagmadUtil.stateless_cmds.ENABLE
+        )
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_no_auth_response_with_mme_restart(self):
+        self._s1ap_wrapper.configUEDevice(1)
+        req = self._s1ap_wrapper.ue_req
+
+        print("************************* Running end-end attach ")
```

Comment:
Can we change the log statement, because test case doesn't really perform end-to-end attach